netlib/http/url.py

```diff
@@ -0,0 +1,96 @@
+import six
```

Comment:
Maybe I'm a bit pendantic here, but shouldn't it be `netlib/url.py`? It's not really HTTP-specific.